### PR TITLE
Add a new hook for using underlays that will prevent default so that …

### PR DESCRIPTION
**Pull request by snowystinger**
…firefox doesn't start text selection

Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-aria/overlays/src/useUnderlay.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
+ * Used when there is an underlay component with the overlay, like a Dialog.
+ */
+export function useUnderlay() {
+  let events;
+  let onPointerDown = (e) => {
```
Comment (ktabors) [inline packages/@react-aria/overlays/src/useUnderlay.ts:18]: Why is onPointerDown outside of the PointerEvent check?
Do we need to also preventDefault on click or press?

**Review by ktabors — CHANGES_REQUESTED** [review]
Shouldn't there be a test or story for this change?

File: packages/@react-aria/overlays/src/useUnderlay.ts (hunk 1 above)
Comment (snowystinger) [reply packages/@react-aria/overlays/src/useUnderlay.ts:18]: Ah don't worry about this PR, it's likely going to be closed as I think #1469 is the way we want to solve this issue
also #1470 will close for the same reason, just waiting on that one to be merged and then I'll handle them

**Comment by snowystinger** [comment]
Closed in favor of #1469

**Pull request closed without merging**